File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/OcsLoadVisibleControlListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
	public class OcsLoadVisibleControlListItemInfo
	{
		private String _inputTab;
		private String _colName;
		private String _visibleYn;

		public String InputTab
		{
			get { return this._inputTab; }
			set { this._inputTab = value; }
		}

		public String ColName
		{
			get { return this._colName; }
			set { this._colName = value; }
		}

		public String VisibleYn
		{
			get { return this._visibleYn; }
			set { this._visibleYn = value; }
		}

		public OcsLoadVisibleControlListItemInfo() { }

		public OcsLoadVisibleControlListItemInfo(String inputTab, String colName, String visibleYn)
		{
			this._inputTab = inputTab;
			this._colName = colName;
			this._visibleYn = visibleYn;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/OcsLoadVisibleControlListItemInfo.java
package nta.med.data.model.ihis.nuro;

public class OcsLoadVisibleControlListItemInfo {
	private String inputTab;
	private String colName;
	private String visibleYn;

	public OcsLoadVisibleControlListItemInfo(String inputTab, String colName,
			String visibleYn) {
		super();
		this.inputTab = inputTab;
		this.colName = colName;
		this.visibleYn = visibleYn;
	}

	public String getInputTab() {
		return inputTab;
	}

	public void setInputTab(String inputTab) {
		this.inputTab = inputTab;
	}

	public String getColName() {
		return colName;
	}

	public void setColName(String colName) {
		this.colName = colName;
	}

	public String getVisibleYn() {
		return visibleYn;
	}

	public void setVisibleYn(String visibleYn) {
		this.visibleYn = visibleYn;
	}

}
